src/auth/newuserwindow.cpp

```diff
@@ -0,0 +1,103 @@
+#include "newuserwindow.h"
+
+#include<QVBoxLayout>
+#include<QLabel>
+#include<QLineEdit>
+#include <QApplication>
+#include <QDesktopWidget>
+#include <QDialogButtonBox>
+#include <QPushButton>
+#include <QDebug>
+#include <QToolTip>
+
```

Comment:
#include <QDebug>
#include <QToolTip>
you don't need it

Reply:
ok